src/suites/cts/validation/create_bind_group_layout.spec.ts

```diff
@@ -0,0 +1,87 @@
+export const description = `
+Create bind group layout validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+function clone(descriptor: GPUBindGroupLayoutDescriptor): GPUBindGroupLayoutDescriptor {
+  return JSON.parse(JSON.stringify(descriptor));
+}
+
+export const g = new TestGroup(ValidationTest);
+
+g.test('some binding index was specified more than once', async t => {
```

Comment:
This is a long name, which could make suppression strings super long, but also helps know what the test does. @kainino0x WDYT?

Reply:
I think it's fine, we can shorten them if it becomes a pain